common/functions-v2/tests/it/scalars/string.rs

```diff
@@ -104,14 +107,54 @@ fn test_to_base64(file: &mut impl Write) {
 fn test_from_base64(file: &mut impl Write) {
     run_ast(file, "from_base64('QWJj')", &[]);
     run_ast(file, "from_base64('MTIz')", &[]);
-    run_ast(file, "to_base64(Null)", &[]);
-    run_ast(file, "to_base64(a)", &[(
+    run_ast(file, "from_base64(Null)", &[]);
+    run_ast(file, "from_base64(a)", &[(
         "a",
         DataType::String,
         build_string_column(&["QWJj", "MTIz"]),
     )])
 }
 
+fn test_quote(file: &mut impl Write) {
+    run_ast(file, "quote('a\0b')", &[]);
+    // run_ast(file, "quote('a\'b')", &[]);
+    run_ast(file, "quote('a\"b')", &[]);
```

Comment:
It seems Parser can not recgonize `\'`.

Reply:
please use raw strings for the following test cases for quote(), because they all unexpectedly unescpaed by rust.

Reply:
Added for other test cases.